portfolio/src/main/webapp/style.css

```diff
@@ -384,4 +384,26 @@ body {
 
     }
 
+}
+
+#message-div {
+
+    width: 100%;
+    margin-top: 50px;
+    margin-bottom: 50px;
+
+}
+
+#message-zone {
+
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+
+}
+
+.message-text{
```

Comment:
I forgot to erase this empty class. I will erase it for the next commit.

Reply:
I appreciate you communicating this! Sounds good